src/components/Showcase/StoryCard.js

```diff
@@ -82,14 +91,7 @@ function StoryCard({ story, ...props }) {
   return (
     <Card className={classes.root}>
       <A href={storyLink} className={classes.cardLink}>
-        <CardActionArea
-          style={{
-            display: 'flex',
-            alignItems: 'flex-end',
-            flexFlow: 'column',
-            height: '100%'
-          }}
-        >
+        <Grid className={classes.CardActionArea}>
```

Comment:
We need `CardActionArea` which `ButtonBase` to give the card the clickable feel.